corehq/apps/reports/views.py

```diff
@@ -194,7 +194,7 @@
 from corehq.apps.hqcase.utils import resave_case
 from corehq.apps.hqwebapp.decorators import (
     use_jquery_ui,
-    use_select2,
+    use_select2_v4,
```

Comment:
Doesn't look like this is defined in this PR. Did it exist before?

Reply:
yes: https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/hqwebapp/decorators.py#L23